eclide/EclDlg.h

```diff
@@ -118,9 +120,25 @@ class CEclDlgQBImpl:
         CString copyStr, titleStr;
         GetTitle(titleStr);
         titleStr.TrimLeft(_T("*"));
+
         copyStr.Format(_T("&Copy \"%s\"\tCtrl+Alt+C"), titleStr);
         m.ModifyMenu(ID_REPOSITORY_COPY, MF_BYCOMMAND | MF_STRING, ID_REPOSITORY_COPY, copyStr);
+
+        IAttribute *attr = m_view.GetAttribute();
+        if (attr) {
```

Comment:
Test if attr is an instance of "IDiskAttribute" and if true use the "GetPath" function:
```
    if (CComQIPtr<IDiskAttribute> diskAttr = attr) {
        ... diskAttr->GetPath()); ...
    }
```

Reply:
You can use the same test to disable the copy path option when not a DiskAttr?